editor/editor_properties_array_dict.cpp

```diff
@@ -645,8 +643,6 @@ bool EditorPropertyArray::can_drop_data_fw(const Point2 &p_point, const Variant
 }
 
 void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) {
-	ERR_FAIL_COND(!_is_drop_valid(p_data));
```

Comment:
Why remove this? The method shouldn't get called if the drop is invalid, but if it does, that's a bug. Wouldn't we want an error message instead of weird and potential undefined behaviour?

Reply:
`can_drop_data_fw()` ensures that the drop is always valid. It's handled by the viewport and if something was wrong, it would affect all drop methods, not just this one. Also `_is_drop_valid()` is relatively expensive, so it's better to avoid calling it redundantly.